File: front/src/models/Imc.ts
import { Aluno } from "./Aluno";

export interface Imc {
  imcId?: string;
  peso: string;
  altura: string;
  imcs: string;
  status?: string;
  criadoEm?: string;
  alunoId: string;
  nome: string;
}

File: MySolution/Prova/Models/Imc.cs
namespace API.Models;

public class Imc
{
    public string ImcId { get; set; } = Guid.NewGuid().ToString();
    public double Peso { get; set; }
    public double Altura { get; set; }
    public double Imcs { get; set; }
    public DateTime CriadoEm { get; set; } = DateTime.Now;
    public int AlunoId { get; set; }
}